packages/dialog/src/index.tsx

```diff
@@ -92,6 +93,7 @@ const DialogInner = forwardRef<HTMLDivElement, DialogProps>(
       onDismiss = noop,
       onMouseDown,
       onKeyDown,
+      ariaHideAncestors,
```

Comment:
I think we'd want to default to true here as well since a [DialogOverlay can be used on its own][1]

[1]: https://reacttraining.com/reach-ui/dialog/#dialogoverlay

Reply:
Thanks. Missed that.